media/css/mozorg/home/home.scss

```diff
@@ -397,13 +388,14 @@ main {
     }
 }
 
-// float layout for wider viewports
+// flex/float layout for wider viewports
 @media #{$mq-tablet} {
-    #who-innovate-wrapper {
+    #impact-innovate-wrapper {
         @include clearfix;
+        @include flexbox;
```

Comment:
This is totally optional, but would another solution to be to use `display: table;` here and then `display: table-cell` for each column (dropping the `float`s)? This may arguably be a little dirty, but it has much wider support than Flexbox?